src/metabase/api/table.clj

```diff
@@ -338,7 +338,7 @@
   "Schema name to use for the saved questions virtual database for Cards that are in the root collection (i.e., not in
   any collection)."
   []
-  (tru "Everything else"))
+  "Everything else")
```

Comment:
For future reference, this is not a user-facing string, and the fact that it was being translated on the BE but not translated on the FE was causing a Cypress test to fail. [Here's a discussion](https://metaboat.slack.com/archives/C010L1Z4F9S/p1666210591030479) about it on Slack.